### DISCUSSION TITLE
frp-all.top 域名会被奇安信识别成 `Gh0st家族后门攻击事件`

### DISCUSSION THREAD

**Discussion by nerdneilsfield** [natfrp/wiki#515] [使用问题]
如图，校园网装了奇安信的安全审计，frp-all.top 的域名会被识别成 `Gh0st家族后门攻击事件` 的 url，然后导致设备被封禁。

![image](https://github.com/user-attachments/assets/3c7107ba-625f-4e94-b1a0-9aa6a0120e5e)

**Comment by fakeboboliu** [comment]
我们并未在相关节点流量中发现 Gh0st 木马，因此很悲伤的事情是，我们对于奇安信的看法无法做任何事，
因为内网穿透服务本身的便捷性，我们只能通过严格的 KYC 来保障安全，但无法完全排除有人盗用他人身份或头铁使用自己的实名进行恶意操作的情况，
如果您发现恶意流量来源，欢迎告知我们。

**Comment by dbsan2333** [comment]
frp-box.top疑似也被我们机房风控了🥲
